packages/sanity/src/desk/panes/document/timeline/timelineItem.styled.tsx

```diff
@@ -35,71 +28,28 @@ export const IconWrapper = styled(Flex)(({theme}: IconWrapperProps) => {
   `
 })
 
-export const Root = styled(MenuItem)(({state = 'enabled', isHovered, theme}: TimelineItemProps) => {
-  const {color} = theme.sanity
-
-  const selectedState = color.button.default.primary.enabled
+export const Root = styled(Button)(({$selected}: {$selected: boolean}) => {
   return css`
     position: relative;
-    min-width: 244px;
-
-    ${state === 'selected' &&
-    css`
-      --card-bg-color: ${selectedState.bg};
-      --card-fg-color: ${selectedState.fg};
-      --card-muted-fg-color: ${selectedState.muted};
-      --card-border-color: ${selectedState.bg};
-      &:not([data-selection-bottom='true']) {
-        border-top-left-radius: 0;
-        border-top-right-radius: 0;
-      }
-    `}
-
-    ${state === 'withinSelection' &&
-    css`
-      border-bottom-left-radius: 0;
-      border-bottom-right-radius: 0;
-      box-shadow: 0px 3px 0px 0px var(--card-bg-color);
-      &:not([data-selection-top='true']) {
-        border-radius: 0;
-      }
-
-      ${IconWrapper} {
-        &::before {
-          background: var(--card-hairline-soft-color);
-        }
-      }
-    `}
-
-      ${state === 'disabled' &&
-    css`
-      [data-ui='Avatar'] {
-        opacity: 0.2;
-      }
-    `}
```

Comment:
We no longer display the 'withinSelection' range, which simplifies this massively – this also means we don't need to render transactions in the changes panel that users cannot interact with.